### PR TITLE
Revert rails 6.1 migrations

### PR DESCRIPTION
**Pull request by LindseySaari**
## Description of change
These migrations aren't available until Rails 6.1. Rolling back and deleting until 6.1 

## Original issue(s)
department-of-veterans-affairs/va.gov-team#0000

### CODE REVIEW

**Review by bramleyjl — DISMISSED** [review]

File: db/schema.rb [hunk 1]
```diff
@@ -10,10 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-
-ActiveRecord::Schema.define(version: 2021_05_25_183405) do
-
-
+ActiveRecord::Schema.define(version: 2021_05_10_202141) do
```
Comment (rileyanderson) [inline db/schema.rb:13]: This needs to be the schema version that is currently in master. There was another change after yours `2021_05_25_183405`

**Review by rileyanderson — CHANGES_REQUESTED** [review]
I think we need to fix the schema version. It should stay the same since there was a change after yours

**Review by rileyanderson — APPROVED** [review]
👍

File: db/schema.rb (hunk 1 above)
Comment (LindseySaari) [reply db/schema.rb]: Oopsie copy/paste error, fixed ✔️

**Pull request merged**